gluoncv/model_zoo/model_zoo.py

```diff
@@ -369,7 +369,14 @@
     'hrnet_w18_small_v2_s' : hrnet_w18_small_v2_s,
     'hrnet_w48_s' : hrnet_w48_s,
     'siamrpn_alexnet_v2_otb15': siamrpn_alexnet_v2_otb15,
-    'monodepth2_resnet18_kitti_stereo_640x192': get_monodepth2_resnet18_kitti_stereo_640x192
+    'monodepth2_resnet18_kitti_stereo_640x192': get_monodepth2_resnet18_kitti_stereo_640x192,
+    'monodepth2_resnet18_kitti_mono_640x192': get_monodepth2_resnet18_kitti_mono_640x192,
+    'monodepth2_resnet18_kitti_mono+stereo_640x192':
+        get_monodepth2_resnet18_kitti_mono_stereo_640x192,
+    "monodepth2_resnet18_posenet_kitti_mono_640x192":
```

Comment:
Can you make double quote to single quote for consistency?

Reply:
Modified, thanks for your reminder!